### ISSUE TITLE
bundler_setup.html description inaccurate

### ISSUE THREAD

**Issue by dgutov** [bundler/bundler-site#130]
http://bundler.io/bundler_setup.html says:

```
Bundler.require(:default)
```

...is exactly equivalent to:

```
...
require 'rack-cache'
...
```

...To tell bundler to use require 'rack/cache', update your Gemfile:

```
...
gem 'rack-cache', :require => 'rack/cache'
...
```

(end of quote)

This doesn't look accurate to me: nobody needs to do that. Even if `rack-cache` didn't include `lib/rack-cache.rb` as a fallback, `Bundle.require(:default)` would require it correctly. At least, Bundler 1.6.2 does.

**Comment by EduardoBautista** [comment]
OK, so it still says:

>Astute readers will notice that the correct way to require the rack-cache gem is require 'rack/cache', not require 'rack-cache'. To tell bundler to use require 'rack/cache', update your Gemfile:
source 'https://rubygems.org'

>```ruby
gem 'rails', '3.0.0.rc'
gem 'rack-cache', :require => 'rack/cache'
gem 'nokogiri', '~> 1.4.2'
```

@dgutov's point still stands that this is not necessarily required when using `Bundle.require(:default)`.  But since this looks to have been reworded, it appears that this part is still desired in the docs.  If you think it should be removed I can send the PR, otherwise this can be closed unless I misunderstood something of course.

**Comment by TimMoore** [comment]
Hmm maybe this specific example isn't accurate, but we do still want to document how to configure the path to require, because some other gems still don't fit the pattern.

I ran across one recently: [`actionform`](http://stackoverflow.com/questions/28709575/rails-doesnt-load-actionform-gem-from-gemfile), however this gem is still unreleased and the maintainers have [asked people not to use it yet](https://github.com/rails/actionform/issues/37).

Maybe an artificial example would be best. At this point, Bundler is popular enough that most gem authors will probably add workarounds to make sure that `Bundler.require` works by default. We should still document this setting for the rare ones that don't.

**Comment by indirect** [comment]
@EduardoBautista the comment is actually about a different problem: Bundler used to try to require `rack-cache` instead of `rack/cache`. That problem has been solved, and that's why it's no longer needed. `Bundler.require` doesn't actually matter here.

If you'd like an example of using the `:require => false` option, you could use the `airbrussh` gem.

**Comment by arbonap** [comment]
👋🏽 hey all, I'd be happy to pick up this doc update and submit a PR to edit out the `Bundler.require(:default)` part.

**Comment by colby-swandale** [comment]
Fixed in #350
